.travis.yml

```diff
@@ -1,12 +1,19 @@
 sudo: false
 
-rvm:
-  - 2.3.0
-  - 2.2
-  - 2.1
-  - 2.0
-  - 1.9.3
-  - jruby-1.7.12
+matrix:
+  include:
+    - rvm: 2.3.0
```

Comment:
Previously just 2.3 did not work on Travis https://github.com/cucumber/cucumber-ruby/commit/d688d886a60dc51973c2fb572a277be4b9720b3a.